### PR TITLE
👷 Add Jenkinsfile for building on Wendy

### PR DESCRIPTION
**Pull request by grdryn**

### CODE REVIEW

**Comment by grdryn** [comment]
Retest this please.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode {
```
Comment (mikenairn) [inline Jenkinsfile]: I wonder if we should consider updating the labels on our node4 slave and setting it explicitly in the Jenkinsfiles. Just a thought.

Comment (grdryn) [reply Jenkinsfile]: @mikenairn Yeah that sounds like a good idea to me, the more explicit the better. After our quick chat, I'm going to change this to not use fhBuildNode at all, and see how that looks/works.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+node('nodejs-ubuntu') {
```
Comment (grdryn) [inline Jenkinsfile]: @mikenairn The `WsCleanup` step from https://github.com/grdryn/fh-pipeline-library/blob/master/vars/fhBuildNode.groovy#L8, is that necessary?

Comment (mikenairn) [reply Jenkinsfile]: @grdryn Not strictly required when we are using kubernetes pods, but having it there ensures it will work correctly if we didn't.

**Pull request merged**